scripts/test-utils.sh

```diff
@@ -7,12 +7,14 @@
 get_latest_snapshot() {
   BRANCH=""
   LATEST_SNAPSHOT=""
+  SNAPSHOT_RELEASE=${SNAPSHOT_RELEASE:=2.10}
```

Comment:
we can bump this per release, but essentially this allows you to add a particular snapshot to pick up, 2.9 , 2.10 etc